fluster/main.py

```diff
@@ -366,23 +343,13 @@ def _add_download_cmd(self, subparsers: Any) -> None:
             type=int,
             default=1,
         )
-        subparser.add_argument(
-            "testsuites", help="list of testsuites to download", nargs="*"
-        )
+        subparser.add_argument("testsuites", help="list of testsuites to download", nargs="*")
         subparser.set_defaults(func=self._download_cmd)
 
     @staticmethod
     def _list_cmd(args: Any, fluster: Fluster) -> None:
-        fluster.list_test_suites(
-            show_test_vectors=args.testvectors,
-            test_suites=args.testsuites,
-            codec=args.codec,
-        )
-        fluster.list_decoders(
-            check=args.check,
-            verbose=args.verbose,
-            codec=args.codec,
-        )
+        fluster.list_test_suites(show_test_vectors=args.testvectors, test_suites=args.testsuites)
```

Comment:
Is "`codec=args.codec,`" missing, or is not necessary anymore?
Also in line 352.